Formula/t/testdisk.rb

```diff
@@ -32,10 +32,7 @@ class Testdisk < Formula
   end
 
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
+    system "./configure", "--disable-silent-rules", *std_configure_args.reject { |s| s["disable-debug"] }
```

Comment:
Does it now fail with `--disable-debug`?

Reply:
it does not fail, but it is not a valid config option.